static/js/components/GcnSelectionForm.jsx

```diff
@@ -81,12 +81,15 @@ const GcnSelectionForm = ({ gcnEvent, setSelectedLocalizationName }) => {
     displayOptionsDefault
   );
 
-  const defaultStartDate = dayjs(gcnEvent?.dateobs).format(
-    "YYYY-MM-DDTHH:mm:ssZ"
-  );
-  const defaultEndDate = dayjs(gcnEvent?.dateobs)
+  const defaultStartDate = dayjs
+    .utc(gcnEvent?.dateobs)
+    .utc()
+    .format("YYYY-MM-DD HH:mm:ss");
```

Comment:
This double `utc()` seems wrong but I'm not sure.

Reply:
It was just unnecessary, yeah.